src/util_libc.rs

```diff
@@ -38,6 +38,8 @@ cfg_if! {
 cfg_if! {
     if #[cfg(target_os = "vxworks")] {
         use libc::errnoGet as get_errno;
+    } else if #[cfg(target_os = "nto")] {
```

Comment:
errno, errno_location, ___errno, etc. all all C functions for other *nix distros.  Any reason you didn't implement this like all other distro's did in the block above?

Reply:
Thanks!